src/components/staking/LiValidator.vue

```diff
@@ -94,10 +94,30 @@ export default {
       type: String,
       /* istanbul ignore next */
       default: () => "returns"
+    },
+    isMultiDenomReward: {
+      type: Boolean,
+      default: false
+    },
+    stakingDenom: {
+      type: String,
+      default: ""
     }
   },
   methods: {
-    percent
+    percent,
+    shortDecimals,
+    selectMostRelevantReward(rewards) {
+      rewards.sort((a, b) => b.amount - a.amount)
+      // we return the reward with the highest amount
+      return shortDecimals(rewards[0].amount)
+        .toString()
+        .concat(
+          this.isMultiDenomReward
+            ? ` ${rewards[0].denom.slice(-3).toUpperCase()}`
```

Comment:
who is TOM 😉 

i don't think this is good logic. we don't need to change the denom.

<img width="191" alt="Screen Shot 2020-02-14 at 8 42 32 AM" src="https://user-images.githubusercontent.com/6021933/74536388-f9a51d80-4f05-11ea-8f34-c94a5796a02d.png">

Reply:
hahaha. Yes, I also saw this one. It is outdated though. I am not showing now any denom for single-denom networks. It is not needed.